website/content/en/docs/olm-integration/generating-a-csv.md

```diff
@@ -24,24 +24,26 @@ CSV fields (with one exception described [below](#csv-fields)).
 1. Path to the Operator manifests root directory. By default `generate csv`
 extracts manifests from files in `deploy/` for the following kinds and adds
 them to the CSV. Use the `--deploy-dir` flag to change this path.
-    * Roles: `deploy/role.yaml`
-    * ClusterRoles: `deploy/cluster_role.yaml`
-    * Deployments: `deploy/operator.yaml`
-2. Path to a directory containing CustomResourceDefinitions (CRDs) and Custom Resource examples (CRs).
+    - Roles: `deploy/role.yaml`
+    - ClusterRoles: `deploy/cluster_role.yaml`
+    - Deployments: `deploy/operator.yaml`
+1. Path to a directory containing CustomResourceDefinitions (CRDs) and Custom Resource examples (CRs).
 Use the `--crd-dir` flag to change this path.
-    * Custom Resources: `deploy/crds/<full group>_<version>_<kind>_cr.yaml`
-    * CustomResourceDefinitions: `deploy/crds/<full group>_<resource>_crd.yaml`
-3. Path to the API types root directory. The CSV generator
+    - Custom Resources: `deploy/crds/<full group>_<version>_<kind>_cr.yaml`
+    - CustomResourceDefinitions: `deploy/crds/<full group>_<resource>_crd.yaml`
+1. _For Go Operators only:_ Path to the API types root directory. The CSV generator
```

Comment:
Is not the gen csv just supported for GO? Why add it so?

Reply:
It was removed in #2989 when it shouldn't have been. No API parsing is done for non-Go operators. Non-Go operators aren't even supported by `generate csv`, although they can use the command.